libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -328,10 +343,7 @@ public void AddRules(IEnumerable<IRule> rules)
             }
         }
 
-        public IDialog FindDialog(string dialogId)
-        {
-            return dialogs.Find(dialogId);
-        }
+        public IDialog FindDialog(string dialogId) => dialogs.Find(dialogId);
```

Comment:
>log(string dialogId) => dialogs.Find(dialogId); [](start = 30, length = 47)

why do you do this?

Reply:
Because the linting rules for the SDK flag not doing this as an error.